### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by MCribbin**

### CODE REVIEW

File: public/coffee/ide/human-readable-logs/HumanReadableLogsRules.coffee [hunk 1]
```diff
@@ -88,4 +88,52 @@ define -> [
...
+		regexToMatch: /LaTeX Error: Something's wrong--perhaps a missing \\item/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/LaTeX_Error:_Something%27s_wrong--perhaps_a_missing_%5Citem"
+		humanReadableHint: """
+			There are no entries found in a list you have created. Make sure you label list entries using the \\item command, and that you have not used a list inside a table incorrectly.
+		"""
+	,
+		regexToMatch: /Misplaced \\noalign/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/Misplaced_%5Cnoalign"
+		humanReadableHint: """
+			You have used a \\hline command in the wrong place.
```
Comment (jpallen) [inline public/coffee/ide/human-readable-logs/HumanReadableLogsRules.coffee]: Can we give this a little more context, e.g. 'probably outside of a table', or something like that?

File: public/coffee/ide/human-readable-logs/HumanReadableLogsRules.coffee [hunk 2]
```diff
@@ -88,4 +88,52 @@ define -> [
...
+		regexToMatch: /LaTeX Error: Environment .+ undefined/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors%2FLaTeX%20Error%3A%20Environment%20XXX%20undefined"
+		humanReadableHint: """
+			You have created an environment (using \\begin{\u2026} and \\end{\u2026} commands) which is not recognized. Make sure you have included the required package for that environment in your preamble, and that the environment is spelled correctly.
+		"""
+	,
+		regexToMatch: /LaTeX Error: Something's wrong--perhaps a missing \\item/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/LaTeX_Error:_Something%27s_wrong--perhaps_a_missing_%5Citem"
+		humanReadableHint: """
+			There are no entries found in a list you have created. Make sure you label list entries using the \\item command, and that you have not used a list inside a table incorrectly.
```
Comment (jpallen) [inline public/coffee/ide/human-readable-logs/HumanReadableLogsRules.coffee]: Can lists be used in tables at all? If there's a correct way to do it, is it possible to say what here?

File: public/coffee/ide/human-readable-logs/HumanReadableLogsRules.coffee [hunk 3]
```diff
@@ -88,4 +88,52 @@ define -> [
...
+		regexToMatch: /Missing \\right inserted/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/Missing_%5Cright_insertede"
+		humanReadableHint: """
+			You have started an expression with a \\left command, but have not included a corresponding \\right command. Make sure that your \\left and \\right commands balance everywhere, or else try using \\Biggl and \\Biggr commands instead as shown <a target=\"_blank\" href=\"https://www.sharelatex.com/learn/Errors/Missing_%5Cright_inserted\">here</a>.
+		"""
+	,
+		regexToMatch: /Double superscript/
+		extraInfoURL: "https://www.sharelatex.com/learn/Errors/Double_superscript"
+		humanReadableHint: """
+			You have written a double superscript incorrectly as a^b^c, or else you have written a prime with a superscript. Remember to include { and } when using multiple superscripts.
```
Comment (jpallen) [inline public/coffee/ide/human-readable-logs/HumanReadableLogsRules.coffee]: Include a quick example for this and subscript, "e.g. a_{b_c}".

**Comment by briangough** [comment]
This was merged in patch-3
https://github.com/sharelatex/web-sharelatex/pull/301

**Pull request closed without merging**